declarations/Facebook.json

```
@@ -15,12 +15,15 @@
       "executeClientScripts": true
     },
     "Privacy Policy": {
-      "fetch": "https://www.facebook.com/about/privacy",
-      "filter": [
-        "removeReturnToTopButtons",
-        "cleanUrls"
+      "fetch": "https://mbasic.facebook.com/privacy/policy/printable/",
+      "select": [
+        "#root"
       ],
-      "select": "._5tkp"
+      "remove": [
+        ".k",
+        ".l",
+        ".bd"
+      ]
```

I was wrong, the selector to remove the "return to the top" elements generate an empty link in the version and this is not good. It should be updated with `.bc`

Another improvement, we could remove unecessary images with `img[style=\"display:none\"]`

```suggestion
        ".bc",
        "img[style=\"display:none\"]"
      ]
```